Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx

```diff
@@ -21,17 +21,22 @@ const HomeContests = () => {
         })();
     }, [ getForHome ]);
 
+    const contestStatusArr = useMemo(
```

Comment:
why is this needed?

Reply:
I need to pass the contest status enumeration value as integer to the contests list component. Since ContestStatus is a string enumeration, I could not find any other way to convert its string values to integer :/

Reply:
Why not just change the type contestStatus?: ContestStatus ? This way the class that is generated in seeAllButtonClassName    will contain a number

Reply:
When I change it to ContestStatus, it just takes the string value.